api/chart_templates.py

```diff
@@ -0,0 +1,137 @@
+import copy
+
+def get_base_pie_chart_config():
+    return {
+        'type': 'pie',
+        'data': {
+            'labels': [],  # filled dynamically
+            'datasets': [{
+                'data': [],  # filled dynamically
```

Comment:
Consider adding a **#filled dynamically** marked fields as parameters of this function, because from the name of the function I would say that it should return some constant pre-defined/pre-fetched config. If you just accept the **labels, data, text** as parameters - it would make this code much more intuitive

Reply:
This can be improved later